symfony/client/src/orangehrmAttendancePlugin/components/RecordAttendance.vue

```diff
@@ -222,7 +221,9 @@ export default {
             date: this.attendanceRecord.date,
             time: this.attendanceRecord.time,
             note: this.attendanceRecord.note,
-            timezoneOffset: this.attendanceRecord.timezone?._offset ?? tzOffset,
+            timezoneOffset:
+              this.attendanceRecord.timezone?._offset ?? timezone.offset,
+            timezoneName: this.attendanceRecord.timezone?.id ?? timezone.name,
```

Comment:
`this.attendanceRecord.timezone?.id` :thinking:

Reply:
![Screenshot from 2022-03-03 09-35-33](https://user-images.githubusercontent.com/91882383/156494536-48c04c70-e619-4650-aaa0-46736a6eb0bb.png)

since we are not passing an id to the autocomplete, timezone name(identifier) is the one used as unique id as discussed with chamara